targets/PythonSdk/make.js

```diff
@@ -99,9 +104,9 @@ function getDeprecationAttribute(tabbing, apiObj) {
     var isError = isDeprecated && (new Date() > deprecationTime) ? "true" : "false";
 
     if (isDeprecated && apiObj.deprecation.ReplacedBy != null)
-        return tabbing + "# [Obsolete(\"Use '" + apiObj.deprecation.ReplacedBy + "' instead\", " + isError + ")]\n";
+        return tabbing + cPythonLineComment + tabbing + "# [Obsolete(\"Use '" + apiObj.deprecation.ReplacedBy + "' instead\", " + isError + ")]\n" + tabbing + cPythonLineComment;
```

Comment:
Python has a proper deprecation meta-decorator which is entirely different than a commented C# deprecation.

Reply:
I'll log a Task. There seems to be multiple ways to do this:
https://pypi.org/project/Python-Deprecated/

https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically